vnext/Desktop/Modules/WebSocketModule.cpp

```diff
@@ -150,12 +149,29 @@ void WebSocketModule::SendEvent(string &&eventName, dynamic &&args) {
 }
 
 // clang-format off
-std::shared_ptr<IWebSocketResource> WebSocketModule::GetOrCreateWebSocket(int64_t id, string&& url)
+shared_ptr<IWebSocketResource> WebSocketModule::GetOrCreateWebSocket(int64_t id, string&& url)
 {
   auto itr = m_webSockets.find(id);
   if (itr == m_webSockets.end())
   {
-    auto ws = m_resourceFactory(std::move(url), /*legacyImplementation*/ false, /*acceptSelfSigned*/ false);
+    shared_ptr<IWebSocketResource> ws;
+    try
+    {
+      ws = m_resourceFactory(std::move(url));
+    }
+    catch (const winrt::hresult_error& e)
+    {
+      SendEvent("webSocketFailed", dynamic::object("id", id)("message", Utf16ToUtf8(e.message())));
```

Comment:
>e [](start = 84, length = 1)

(nit) not sure how useful, but having actual hresult int value could be of value